docs/user-accounts.md

```diff
@@ -0,0 +1,94 @@
+# User accounts
+
+ReportMI consists of two services that are used by two distinct sets of users:
+
+- The admin app (hereafter RMI Admin), which is the service contained in this
+  repository. This is used by Crown Commercial Service (CCS) staff to manage
+  users, suppliers, and process returns. It also provides an API that's consumed
+  by the frontend.
+
+- The frontend (hereafter RMI Frontend), which is used by external suppliers to
+  submit their returns to CCS.
+
+In production, CCS staff log into RMI Admin using their Google account. They use
+RMI Admin to manage logins for external suppliers, which are stored in Auth0.
+Suppliers log into RMI Frontend, which talks to the RMI Admin API and to Auth0.
+
+          +--------------------+        +-------------------+
+          |     Google SSO     |        |       Auth0       |
+          | (RMI Admin logins) |        | (Supplier logins) |
+          +--------------------+        +-------------------+
+                      ^                       ^     ^
+                      |                       |     |
+                      |     +-----------------+     |
+                      |     |                       |
+                  +-----------+               +--------------+
+                  | RMI Admin |<--------------| RMI Frontend |
+                  +-----------+               +--------------+
+                        ^                             ^
+                        |                             |
+                    CCS staff                     Suppliers
+
+
+## RMI Admin
+
+In production, CCS staff log into RMI Admin using their Google SSO credentials.
+The set of users that are allowed access to the app is stored in the environment
+variable `ADMIN_EMAILS`.
+
+In development, the environment variable `ADMIN_EMAILS` is still used but no
+authentication is required. Just add your email address to `ADMIN_EMAILS` in
+`docker-compose.env` before starting the app, and you will be able to log in
+using that address.
+
+
+## RMI Frontend
+
+This is the app used by external suppliers to submit their returns. Logins to
+this service are authorized by the RMI Admin API, and supplier login details are
+stored in Auth0.
```

Comment:
Do you think it would be useful to have a few-words description of Auth0? I had never heard of it before. And when I Google it, the snippet from the official page is hardly expository: "Auth0 is the solution you need for web, mobile, IoT, and internal applications. Loved by developers and trusted by enterprises."

Reply:
I agree it would be nice to have more detail on the wider SSO strategy in use, what standards are used and how Auth0 fits in as a service. That said I presume that these are James' notes that he's looking to helpfully contribute to help the next person and may no longer have the time to take it this step further. I think it's important to ensure we get the contribution we have committed and look to build on it later rather than leave the value here stuck in limbo.

Reply:
:+1: - makes sense; let's get it merged!